### ISSUE TITLE
Installed but see nothing

### ISSUE THREAD

**Issue by jpfieber** [karstenpedersen/obsidian-daily-note-navbar#1]
I installed the plugin on the latest version of Obsidian on Windows. I enabled the plugin but don't see the navigation bar anywhere.  Closed and reopened Obsidian, still nothing.  I don't see any errors in the console that reference the plugin.  Am I missing a step?

**Comment by karstenpedersen** [comment]
Hi, thanks for the issue :)

I have tested it on my Windows machine, and I can see the bar:

![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/53978294/0cb9ae26-d121-4399-9625-9c09bea6332c)

The navbar should only be visible from within daily notes. If you are in daily notes, and still can't see it, then it might be one of these things:
- What is the name of your daily notes? (the plugin might not recognize your daily note format)
- Are you using a custom theme or css files? (the plugin inserts the bar if it can find a html element, based in a css selector)

**Comment by jpfieber** [comment]
My daily notes are named YYYY/YYYY-MM/YYYY-MM-DD_ddd.  I'm using the 'Minimal' theme. Only a few custom CSS based around showing icons on links.
I was wrong, there is a related error in the console:
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/12567659/f7f43d52-f76a-4371-8b5f-ea124fa9db69)

**Comment by karstenpedersen** [comment]
Yeah that is what I thought. My function fails to get the date from the file name, because I check for another date format.

I think I can fix this by adding a field in settings that is the format of daily note files.

**Comment by karstenpedersen** [comment]
Okay tested it a bit, and I don't think that it is that. 

When I use the [Minimal](https://github.com/kepano/obsidian-minimal) theme I have to hover over the navbar at the top of the file to see it. I think the theme hides it.

I will still add a field to specify the date format of daily note files, as the warning you are seeing shouldn't be there.

If it is not the theme that hides it, then I probably need some more information.

**Comment by karstenpedersen** [comment]
If that is the problem then you can override it using this:

```css
.view-header-title-container {
  opacity: 1;
}
```

**Comment by jpfieber** [comment]
I also tried the themes 'Atom' and 'Thinks' and it didn't show up in either of those. I tried adding your snippet but it didn't help.  I always work in "Preview" mode, I hope it's meant to work there.

**Comment by jpfieber** [comment]
I turned off all my snippets, made no difference.

**Comment by karstenpedersen** [comment]
Can you try opening the inspector using "Ctrl+Shift+I" (works for me) and use the element picker on where the navbar should be and send a screenshot of the html.

**Comment by karstenpedersen** [comment]
I am on version 1.5.8 of Obsidian, but I don't think they have updated the classes so my selectors are broken.

**Comment by jpfieber** [comment]
I'm also on 1.5.8
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/12567659/d8ed8028-224b-4f5a-ad3b-217a4f2fe259)

**Comment by jpfieber** [comment]
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/12567659/dfebdaaf-b4a1-4970-b605-c020ee86196f)

**Comment by karstenpedersen** [comment]
My HTML is like this. I have pointed at the div around the location where you can edit the filename. 
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/53978294/e46d80b9-2e12-41ee-ac1a-0ae6fe4d54f6)

**Comment by karstenpedersen** [comment]
It looks like your classes are different, so I think that is why the plugin doesn't work, but I don't know why that is.

**Comment by karstenpedersen** [comment]
I think what you have highlighted is a H1 maybe, and that might be why our classes aren't the same.

The plugin places the navbar where you usually rename files in the top center of the view.

![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/53978294/5dfd937b-d962-401b-bfd0-405a3b5dcf6d)

**Comment by jpfieber** [comment]
Here's that part of the code expanded:
![image](https://github.com/karstenpedersen/obsidian-daily-note-navbar/assets/12567659/0a24a92b-bcc7-4940-afcf-4d73d491c59c)

**Comment by karstenpedersen** [comment]
Okay, I have released a pre-release where you can define the date format of the filename of daily notes.

https://github.com/karstenpedersen/obsidian-daily-note-navbar/releases/tag/0.1.1

You can use https://github.com/TfTHacker/obsidian42-brat if you want to try it out.

I will not look at this more today, but feel free to try it out when you have time and see if it fixes your issue :)

**Comment by jpfieber** [comment]
I wasn't able to get BRAT to install it (says it doesn't have a manifest.json even though it does), so I manually copied the files. It WORKS! Even before I adjusted the "Daily note date format" setting that you added, it seems to do what it's supposed to (and if I change to YYYY-MM-DD, it still works despite my files being YYYY-MM-DD_ddd).  Thanks, I can remove the navigation from my journal footer now and use this more elegant solution!
